### ISSUE TITLE
Option to make all Steam links open in the client rather than in the web browser

### ISSUE THREAD

**Issue by xCeeTee** [JosefNemec/Playnite#3050]
### Feature description

Adding "steam://openurl/" to the beginning of a Steam url opens the link in the client. It would be amazing if there was an option that does this automatically for the links. And if possible for other clients, although I'm yet to discover if they work.

**Comment by JosefNemec** [comment]
Why would you want to have links open in Steam client instead of normal web browser?

**Comment by xCeeTee** [comment]
The performance for me is a lot better in client so I don't like to use the website.

**Comment by sukarn-m** [comment]
> Why would you want to have links open in Steam client instead of normal web browser?

Niche use case, low priority:

In my case, Steam client remains logged in, while Firefox (the default browser) is set up to use temporary containers that destroy cookies when the tab is closed. At the moment, I have set up a permanent container for Steam for these links in Firefox, but opening them in the Steam client would be preferable.

**Comment by darklinkpower** [comment]
This extension is made with the purpose of opening various sections in the Steam client https://playnite.link/addons.html#Steam_Viewer_0a3edabb-065f-4056-a294-d6bc0656e2ac

Imo the requested feature is too niche and very oriented for a specific store/client so I think if implemented, the feature should be more broad and flexible.

**Comment by sukarn-m** [comment]
> This extension is made with the purpose of opening various sections in the Steam client https://playnite.link/addons.html#Steam_Viewer_0a3edabb-065f-4056-a294-d6bc0656e2ac
> 
> Imo the requested feature is too niche and very oriented for a specific store/client so I think if implemented, the feature should be more broad and flexible.

Fair enough. Agreed on "too niche" and "specific store/client". I'm not convinced myself that the convenience would be worth running a link interpreter/converter on every single link click. An extension could handle this better, if they have the ability to intercept and change links clicked in the library.

Steam Link Viewer (the linked extension) isn't _perfect_ as it requires a right click on the game -> Steam Viewer -> Desired Option, instead of just clicking on the link in the game's details. But, even if it takes a few extra clicks, it does a good job of jumping straight into different parts of the Steam client.
